usaspending_api/api_contracts/contracts/v2/disaster/overview.md

```diff
@@ -0,0 +1,57 @@
+FORMAT: 1A
+HOST: https://api.usaspending.gov
+
+# Disaster Spending Overview [/api/v2/disaster/overview/]
+
+This endpoint provides funding and spending details from emergency/disaster supplemental funding legislation.
+
+## GET
+
++ Response 200 (application/json)
+    + Attributes (object)
+        + `funding` (required, array[Funding], fixed-type)
+        + `total_budget_authority` (required, number)
+        + `spending` (required, Spending, fixed-type)
+
+    + Body
+
+            {
+                "funding": [
+                    {
+                        "def_code": "L",
+                        "amount": 7410000000
+                    },
+                    {
+                        "def_code": "M",
+                        "amount": 11230000000
+                    }
+                ],
+                "total_budget_authority": 2300000000000,
+                "spending": {
```

Comment:
I would recommend putting `total_budget_authority` as a member of `funding`, like:
```
{
    "funding": {
        "total_budget_authority": 2300000000,
        "def_codes": [
            {
                "code": "L",
                "amount": 7410000000
            },
            {
                "code": "M",
                "amount": 11230000000 
            }
        ]
    },
    "spending": {...}
}
```

This also opens up a potential option for a `"loans": {}` object at the same level of funding and and spending.